MainProject/UABCompany.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace MainProject
+{
+    public class UABCompany : Company
+    {
+        private string ownerName;
+
+        public UABCompany(string name, int year, string vatNumber, string ownerName) : base(name, year, vatNumber)
+        {
+            this.ownerName = ownerName;
+        }
+
+        public override void PrintCompanyEmployeesCountByCity(string city)
+        {
+            base.PrintCompanyEmployeesCountByCity(city);
+            Console.WriteLine("Counting done");
+            Console.WriteLine();
+        }
+
+        public override string GetCompanyInfo() => base.GetCompanyInfo() + $" and owner name {ownerName}";
+
+        public new void PrintCompanyName() => Console.WriteLine($"Company hidden name is {Name}");
```

Comment:
I don't get the logic to print message with "hidden". If just to show how hiding works in C# - yes, this example show it, but there is no logic